.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
```

Comment:
Eventually you could update this document to outline how you're using Changesets; any custom commands or steps or plugins you're using or describing why certain configurations were used to provide context.

Reply:
Sure, will do that